src/Header.jsx

```diff
@@ -0,0 +1,9 @@
+export function Header() {
+  return (
+    <>
```

Comment:
fragments are only really needed if we need to wrap multiple tags and they need a parent tag, this component already has a parent tag header so the fragment isn't needed.